Aztec/Classes/TextKit/TextView.swift

```diff
@@ -558,6 +558,12 @@ open class TextView: UITextView {
     }
 
 
+    private func updateTypingAttributes() {
+        if (textStorage.length > 0){
+            let location = max(0,min(selectedRange.location, textStorage.length-1))
```

Comment:
I'd just recommend to add an explanation as to why we're doing `max(min())`, so that when we come back to it we immediately understand the reasons behind this choice.